src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlBulkCopy.cs

```diff
@@ -1175,6 +1180,11 @@ private Task ReadFromRowSourceAsync(CancellationToken cts)
                         throw;
                     }
                 }
+                finally
+                {
+                    _insideRowsCopiedEvent = false;
```

Comment:
Why do we set the `_insideRowsCopiedEvent `?

Reply:
I've reverted that, wasn't intended.